homestead.md

```diff
@@ -588,6 +608,7 @@ php73
 php74
 php80
 php81
+php82
```

Comment:
PHP 8.2 is already available, isn't it?

Reply:
On December 8th: https://php.watch/news/2022/11/php-8.2-release-rescheduled